### PR TITLE
added readme file and whiteboard

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

**Comment by tahminaringer** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [x ] Picture of whiteboard

File: javascript/code-challenges/treeIntersection/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- If no numbers match
+- There is nothing in the the tree
+
+## Approach & Efficiency
+
+I wrote my code first and then wrote tests to validate the code.
+
+### Whiteboard
+
+- [Whiteboard](whiteboard.jpg)
```
Comment (adrianhuebner) [inline javascript/code-challenges/treeIntersection/README.md:32]: You are off to a great start, once you have a chance to study this more, get code and tests written, I will regrade for you!
